blocks/hero/hero.shadow.css

```diff
@@ -0,0 +1,63 @@
+:host {
+  display: grid;
+  position: relative;
+  padding: 32px;
+  min-height: 300px;
+}
+
+:host h1 {
+  max-width: 1200px;
+  margin-left: auto;
+  margin-right: auto;
+  color: white;
+  font-family: var(--heading-font-family);
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h6 {
+  font-family: var(--heading-font-family);
+  font-weight: 600;
+  line-height: 1.25;
+  margin-top: 1em;
+  margin-bottom: 0.5em;
+  scroll-margin: calc(var(--nav-height) + 1em);
+}
+
+h1 {
+  font-size: var(--heading-font-size-xxl);
```

Comment:
wouldn't this be better inherited from the outside or is there no inheritance of styles?into the shadow dom?

Reply:
shadow dom is encapsulated, it does inherit some things like body styles and variables so we need to style inside it